syndeo-template/src/test/java/com/google/cloud/syndeo/it/KafkaToBigQueryIT.java

```diff
@@ -95,14 +98,50 @@ public class KafkaToBigQueryIT {
   public void setUp() {
     bigQueryResourceManager =
         DefaultBigQueryResourceManager.builder("kafka-bq-test", PROJECT).build();
-    bigQueryResourceManager.createDataset(BIGQUERY_DATASET);
+    bigQueryResourceManager.createDataset(REGION);
+  }
+
+  @Test
+  public void testErrorOnCreateNeverIfTableNotExisting() throws Exception {
+    JsonNode rootConfig =
+        KafkaToBigQueryLocalTest.generateConfigurationWithKafkaBootstrap(
+            KAFKA_BOOTSTRAP_SERVER, testName.getMethodName());
+    ((ObjectNode) rootConfig.get("sink").get("configurationParameters"))
+        .put("createDisposition", "CREATE_NEVER");
+    ((ObjectNode) rootConfig.get("sink").get("configurationParameters"))
+        .put(
+            "table",
```

Comment:
nit: shouldn't we just set the "table" by passing this name on ln 108 instead of testName.getMethodName() ?

Reply:
great catch. fixed!